packages/shorebird_code_push_client/lib/src/code_push_client.dart

```diff
@@ -300,6 +300,23 @@ class CodePushClient {
         .toList();
   }
 
+  /// List all collaborators for the provided [appId].
+  Future<List<Collaborator>> getCollaborators({required String appId}) async {
+    final response = await _httpClient.get(
+      Uri.parse('$hostedUri/api/v1/apps/$appId/collaborators'),
```

Comment:
@eseidel I thought about the helper you suggested and I'm not sure it's worth it since it saves us from having to write `$hostedUri/api/v1/` but it becomes less obvious what the endpoint is 🤷